guacamole/src/main/webapp/translations/it.json

```diff
@@ -307,6 +316,7 @@
         "SECTION_HEADER_NETWORK"            : "Network",
         "SECTION_HEADER_PERFORMANCE"        : "Performance",
         "SECTION_HEADER_REMOTEAPP"          : "RemoteApp"
+        "SECTION_HEADER_SFTP" : "SFTP"
```

Comment:
Please test your changes before opening a PR. This is invalid JSON and will not work - the previous line is missing a comma.